core-skills/web-application-development-with-react/README.md

```diff
@@ -125,11 +125,9 @@ The candidate should create a React application which satisfies the following.
   - Candidate should walk assessor through tests which give maintainers confidence 
     that the component submits the data from the form to the API.
 - There should be a JS layer of indirection between the components and the API
-  - Candidate should showcase a class which makes HTTP API requests, and tests which
+  - Candidate should showcase standalone code which makes HTTP API requests, and tests which
     cover this.
-  - Candidate should showcase tests for the component which mocks boundary between the
-    component and model.
-- The application should have component tests should have minimal coupling to the HTML
-  structure they render.
-  - Candidate should showcase their use of test specific attributes in the test suite
+  - Candidate should showcase tests which prove minimal coupling between the component and API interface.
+- The application should have component tests should have minimal coupling to markup they render.
```

Comment:
"_should_ have component tests _should_ have minimal coupling to markup they render"

Dan this makes no sense.